src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -34,12 +34,9 @@
 /**
  * Matcher for the input.
  * @since 0.11
- * @checkstyle IndentationCheck (500 lines)
- * @todo #7:30min All the matchers should implement describeMismatchSafely and
- *  their tests must verify that the implementation of the descriptions
- *  are satisfactory. The matchers should not expose publicly the xxxSafely
- *  method and the tests should rely on actual real use with assertThat.
- *  See ScalarHasValueTest for an example of a satisfactory result.
+ * @todo #7:30min Remove checkstyle suppression when qulice will fix this issue.
```

Comment:
@Iprogrammerr here you need to reference the issue you tackled (#75) and not the original one (#7)